includes/base/widget-base.php

```diff
@@ -362,6 +363,31 @@ protected function get_initial_config() {
 		return array_replace_recursive( parent::get_initial_config(), $config );
 	}
 
+	/**
+	 * Check if widget should be displayed in search result according to widget categories.
+	 *
+	 * @param array $categories Widget categories.
+	 */
+	protected function isWidgetShouldBeHidden( $categories ) {
+		$categories_to_hide = [ 'wordpress', 'theme-elements-archive' ];
```

Comment:
Move the logic to the `theme-elements-archive` widgets

Reply:
Removed. It is not relevant anymore since we do not support archive filter yet.